src/modules/core/apollo/apolloClient.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  ApolloClient,
+  ApolloLink,
+  HttpLink,
+  InMemoryCache,
+  NormalizedCacheObject,
+} from '@apollo/client';
+import merge from 'deepmerge';
+import isEqual from 'lodash.isequal';
+import { useMemo } from 'react';
+
+export const APOLLO_STATE_PROP_NAME = '__APOLLO_STATE__';
+
+export type ApolloState = {
+  apolloState: NormalizedCacheObject;
```

Comment:
Since we're going to use `__APOLLO_STATE__ ` we need to update this accordingly.